catalog/ui/search-ui/search-endpoint/src/test/java/org/codice/ddf/ui/searchui/query/controller/SearchControllerTest.java

```diff
@@ -140,7 +139,7 @@
 
     @Before
     public void setUp() throws Exception {
-        framework = createFramework();
+        framework = createFramework(2);
```

Comment:
@rzwiefel I know and love that feature in IntelliJ, but it is bad practice to assume the development environment when this is an open-source project and the code itself should be universally readable to all. The feature is simply for ease of development, but many people still view the code in raw form in the GitHub text editor, and we should always assume that there are no environment features to help with viewing raw text.